validator/rules_src/__init__.py

```diff
@@ -1,17 +1,20 @@
 import importlib
-import os
 import pkgutil
 from pathlib import Path
+from abc import ABCMeta
 
 
-class Rule:
+class Rule(metaclass=ABCMeta):
     def __init__(self):
         self.error_message = "error"
 
     def __call__(self, arg):
-        return False
+        return self.check(arg)
 
-    def __from_str__(self):
+    def check(self, arg) -> bool:
```

Comment:
can you please remove static type checking? It is good to have them, but we are still in beta and they might be change a lot by time, so we are saving them for later